### ISSUE TITLE
Does the GroupRole type have any recommended role types?

### ISSUE THREAD

**Issue by eneufeld** [FamilySearch/gedcomx#325]
The https://github.com/FamilySearch/gedcomx/blob/master/specifications/conceptual-model-specification.md#322-the-grouprole-data-type has a property `type`. Does this property have any recommended types which should be used as do most of the other occurrences of the type property?

**Comment by stoicflame** [comment]
No, we haven't started developing that controlled vocabulary. I'd be happy to entertain a proposal, though.

For now, there's only [Enumerated Value Extensions](https://github.com/FamilySearch/gedcomx/blob/master/specifications/conceptual-model-specification.md#52-enumerated-value-extensions).

**Comment by nractive** [comment]
> No, we haven't started developing that controlled vocabulary. I'd be happy to entertain a proposal, though.

How (when) does a proposed type in the GEDCOM X Conceptual Model become an official specification? 

As @stoicflame and others already know, my genealogy domain concerns documenting facts of slavery. `Enslavement` is now among the Person Fact Types; I guess that means “official.” In the Conceptual Model there are new types: `EnslavedBy`, `Group`, `GroupRole`, and the `NonConsensual` qualifier, among others of great utility and importance. Do they eventually get rolled into the official specification?

I’m just not knowledgeable about the process.

**Comment by stoicflame** [comment]
It becomes "official" when a proposal is made, the proposal gets accepted, and the proposal is merged into the official spec. Just like it happened at #323.
